pushapkscript/script.py

```diff
@@ -63,12 +64,33 @@ def _get_product_config(context, android_product):
         raise ConfigValidationError('"products" is not part of the configuration')
 
     try:
-        return products[android_product]
-    except KeyError:
+        return [product for product in products if android_product in product['product_names']][0]
+    except IndexError:
         raise TaskVerificationError('Android "{}" does not exist in the configuration of this instance.\
     Are you sure you allowed to push such APK?'.format(android_product))
 
 
+def _get_publish_config(product_config, payload, android_product):
+    if not product_config.get('map_channels_to_apps'):
```

Comment:
Hmm, good question.

* I believe that mapping channels to apps should be the default behaviour
* I feel that the "default" value of blank configuration options is usually `false`, so it's good to to reflect that in how you define your configuration properties
* So, in our case, having the config option be called `map_channels_to_tracks`, so that it acts as an "override", and leaving it blank does the default behaviour

:+1: